### PR TITLE
remove video logs if set to silent

### PR DESCRIPTION
**Pull request by kerkeniakov**
Probably missed something, since this is my first open source contribution. 
Please review :)

### CODE REVIEW

**Comment by kerkeniakov** [comment]
this PR should help with https://github.com/webdriverio-community/wdio-video-reporter/issues/76

File: dist/wdio-video-reporter.js [hunk 1]
```diff
@@ -651,9 +651,11 @@ class Video extends WdioReporter {
         started = true;
         helpers.debugLog(`\n--- FFMPEG is done ---\n\n`);
 
-        this.write('\nGenerated:' + JSON.stringify(this.videos, undefined, 2) + '\n\n');
+        if (config.logLevel === 'silent') ; else {
```
Comment (christian-bromann) [inline dist/wdio-video-reporter.js:654]:
```suggestion
        if (config.logLevel !== 'silent') {
```

File: src/index.js [hunk 2]
```diff
@@ -218,9 +218,13 @@ export default class Video extends WdioReporter {
         started = true;
         helpers.debugLog(`\n--- FFMPEG is done ---\n\n`);
 
-        this.write('\nGenerated:' + JSON.stringify(this.videos, undefined, 2) + '\n\n');
+        if (config.logLevel === 'silent') {
+          // no console output
+        } else {
```
Comment (christian-bromann) [inline src/index.js:223]:
```suggestion
        if (config.logLevel !== 'silent') {
```

**Review by christian-bromann — COMMENTED** [review]
minor code formatting

**Review by christian-bromann — APPROVED** [review]
👍

**Pull request merged**